mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (!order) {
+      return res.status(404).json({ message: "Order does not exist" });
+    }
+    res.status(200).json({ message: "Order deleted successfully" });
+  } catch (err) {
+    res.status(500).json({ error: errorHandler(err) });
```

Comment:
Error handling should be consistent throughout the code. In other parts of the code, error responses are sent with a JSON object containing an 'error' key, but in this case, it's sent with an 'error' key directly. It should be consistent and follow the same pattern as the rest of the code. It should be `res.status(500).json({ error: errorHandler(err), message: 'Failed to delete order' });`